plugins/volumeFilter_test.go

```diff
@@ -1127,6 +1160,16 @@ func runTestVolumeFilterFn(
 func makeSellOpAmtPrice(amount float64, price float64) *txnbuild.ManageSellOffer {
 	return &txnbuild.ManageSellOffer{
 		Buying:  txnbuild.NativeAsset{},
+		Selling: testQuoteAsset,
```

Comment:
we should be selling the base asset here -- base asset should be native. apologies if I indicated otherwise previously